packages/axios-extension/src/abap/types/adt-types.ts

```diff
@@ -57,3 +57,15 @@ export interface AdtCategory {
     term: AdtCategoryTerm;
     scheme: string;
 }
+
+export interface TransportRequest {
+    transportNumber: string;
+    user: string;
+    date: string;
+    time: string;
+    description: string;
+    client: string;
+    targetSystem: string;
+    transportRequestFunction: string;
+    transportRequestStatus: string;
```

Comment:
What are the allowable values here?

Reply:
Removed properties that we are uncertain about their data format.